p/prometheus__client_golang/prometheus__client_golang_ubi_8.5.sh

```diff
@@ -0,0 +1,45 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: github.com/prometheus/client_golang
+# Version	: v1.8.0
+# Source repo	: https://github.com/prometheus/client_golang
+# Tested on	: UBI 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Atharv Phadnis <Atharv.Phadnis@ibm.com>
+#
```

Comment:
looks like the new fields "Language" and "Travis-Check" are missing from the header?

Reply:
Updated, please review